source/sitemap.xml

```diff
@@ -31,7 +31,7 @@
   <changefreq>weekly</changefreq>
 </url>
 <url>
-  <loc>http://developers.virtusize.com/integration-guide/integration.html</loc>
```

Comment:
Should `integration-guide/integration.html` really be removed from the list altogether? The page still exists, no?